test/check-application

```diff
@@ -1419,6 +1426,9 @@ class TestApplication(testlib.MachineCase):
             cpuShares = self.execute(auth, "podman inspect --format '{{.HostConfig.CpuShares}}' busybox-with-tty || podman inspect --format '{{.HostConfig.CPUShares}}' busybox-with-tty").strip()
             self.assertEqual(cpuShares, '512')
 
+            restartPolicy = self.execute(auth, "podman inspect --format '{{.HostConfig.RestartPolicy}}' busybox-with-tty").strip()
```

Comment:
Can you please also add test that on some other container it is not set (or set to `no`) and then also check `always` option? Thanks!